getting_started/build-guide/build-from-scratch.rst

```diff
@@ -448,23 +474,23 @@ Copy them to your target machine's /lib/firmware/intel/sof folder.
 Step 4 Build topology and tools
 ===============================
 
-Build with scripts
-------------------
+One-step rebuild from scratch
+-----------------------------
 
 .. code-block:: bash
 
    cd ~/work/sof/sof/
    ./scripts/build-tools.sh
 
-Build with commands
--------------------
+Incremental build
+-----------------
 
 .. code-block:: bash
 
    cd ~/work/sof/sof/tools/
-   mkdir build_tools && cd build_tools
-   cmake ..
-   make -j4
+   mkdir build_tools
+   cmake -B build_tools
```

Comment:
We shouldn't use `-B` of  cmake's cli, because it's not in our minimum supported version (3.10).

Reply:
Instead of replacing, I added -B as an example with a 3.13 version caveat. 3.13 is 1+ year old now and -B is very convenient so worth knowing.